examples/experimental/i3s-17-and-debug/mesh-layer/mesh-layer.js

```diff
@@ -54,10 +54,10 @@ export default class MeshLayer extends SimpleMeshLayer {
   }
 
   initializeState() {
-    const {pickFeatures, segmentationData} = this.props;
+    const {pickFeatures, objectIds} = this.props;
```

Comment:
I am a proponent of aligning the names of related symbols (unless there are reasons not to): i.e. `pickObjects`/ `objectIds`, or `pickFeatures` / `featureIds`?

I think we talk about `objects` in other places so that may be a good choice.

Reply:
Renamed to `featureIds`